frontend/apps/customer/.env.example

```
@@ -1,10 +1,18 @@
 FEEDBACK_URL=
 TILAVARAUS_API_URL=http://localhost:8000
-PROFILE_UI_URL="https://google.com"
+PROFILE_UI_URL=https://google.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PROFILE_UI_URL key should go before the TILAVARAUS_API_URL key

(UnorderedKey)